drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -533,19 +534,6 @@ virtio_vdpa_dev_notifier(void *arg)
 	}
 	DRV_LOG(INFO, "%s vid %d dev notifier work of vq id:%d finish",
 			work->priv->vdev->device->name, work->priv->vid, work->vq_idx);
-	/* Notify device anyway, in case loss doorbell */
```

Comment:
this might needed if notify comes late, we should kick again

Reply:
This is just for debug, will remove.